www/modules/recalls/views/Search.html

```diff
@@ -1,10 +1,24 @@
 <div class="recalls search" layout="column" layout-fill>
-    <div class="backdrop">
-        <div class="search-results-feedback" layout-fill
-             layout="column" layout-align="center center"
-             ng-if="search.recalls.length == 0">
-            <div i18n="recalls.search.noResults"></div>
-            <md-button scan-button class="md-primary md-raised" i18n="recalls.search.scanButton"></md-button>
```

Comment:
remove scan-button directive here and action should reset search page to blank entry state